### PR TITLE
[pt-br] Add docs/tasks/configure-pod-container/create-hostprocess-pod.md

### PR DESCRIPTION
**Pull request by DonatoHorn**

### CODE REVIEW

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
+---
+title: Crie um Hostprocess do Windows
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:2]: Original title said "Windows HostProcess Pod".
Perhaps "Crie um Pod de HostProcess do Windows" delivers that better?

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+content_type: task
+weight: 20
+min-kubernetes-server-version: 1.23
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+Os contêineres do Windows `HostProcess` permitem que você execute
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:12]: "contêineres de HostProcess do Windows" perhaps?

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+cargas de trabalho em um host Windows contêinerizado. Esses contêineres operam como
+processos normais, mas tem acesso à rede host do namespace,
+armazenamento e dispositivos, quando recebem os privilégios de usuário apropriados.
+Os `HostProcess` contêineres podem ser usados para implantar plugins de rede,
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:16]: Perhaps rewriting it to "contêineres de HostProcess"?

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+configurações de armazenamento, plugins de dispositivos, `kube-proxy`, e outros
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:17]: Suggestion: remove code formatting from here.

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+componentes para nós do Windows, sem a necessidade de `proxies` dedicados ou
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:18]: Stylizing as italic instead of code formatting would be better here.

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+que esteja disponível no host ou esteja no domínio da máquina host, 
+permitindo aos administradores restringirem o acesso aos recursos, 
+por meio de permissões de usuário. Enquanto nenhum sistema de arquivos 
+ou isolamento de processos são suportados, um novo volume é criado no host 
+ao iniciar o contêiner para fornecer um espaço de trabalho limpo e consolidado. 
+Contêineres `HostProcess` também podem ser construídos sobre as imagens base 
+do Windows existente, e não herdam os mesmos
+[requisitos de compatibilidade]
+(https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility)
+como contêineres do `Windows Server`, o que significa que a versão das
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:34]: This link is not rendering correctly due to extra space between `[]` and `()`.

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hello there, @DonatoHorn!

Thanks for your contribution! 🎉 

I wasn't able to fully finish reviewing this page, but found some pieces that need to be addressed. I also feel like a bunch of sentences are not reading naturally where they mention HostProcess. Could you please take another look at those?

Thanks again for your contribution and please don't hesitate to reach out should you have any questions!

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+Os contêineres de HostProcess do Windows permitem que você execute
+cargas de trabalho em um host Windows contêinerizado. Esses contêineres operam como
+processos normais, mas tem acesso à rede host do namespace,
+armazenamento e dispositivos, quando recebem os privilégios de usuário apropriados.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:15]: Minor: "têm acesso ao namespace, armazenamento e dispositivos da rede host" would be more idiomatic.

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 8]
```diff
@@ -0,0 +1,245 @@
...
+Os contêineres HostProcess podem ser usados para implantar plugins de rede,
+configurações de armazenamento, plugins de dispositivos, kube-proxy, e outros
+componentes para nós do Windows, sem a necessidade de *proxies* dedicados ou
+a instalação direta dos serviços de host.
+
+Tarefas administrativas, como a instalação de `patches` de segurança,
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:21]: Suggestion: "instalação de correções de segurança" instead of "`patches` de segurança".

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 9]
```diff
@@ -0,0 +1,245 @@
...
+processos normais, mas tem acesso à rede host do namespace,
+armazenamento e dispositivos, quando recebem os privilégios de usuário apropriados.
+Os contêineres HostProcess podem ser usados para implantar plugins de rede,
+configurações de armazenamento, plugins de dispositivos, kube-proxy, e outros
+componentes para nós do Windows, sem a necessidade de *proxies* dedicados ou
+a instalação direta dos serviços de host.
+
+Tarefas administrativas, como a instalação de `patches` de segurança, 
+coleta de logs de eventos, e mais pode ser realizado sem a necessidade 
+de operadores de cluster, que façam login em cada nó do Windows.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:23]: Rewording suggestion: "de que operadores de cluster façam login em cada nó".

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 10]
```diff
@@ -0,0 +1,245 @@
...
+como contêineres do `Windows Server`, o que significa que a versão das 
+imagens base não precisa combinar com a do host. É, no entanto, 
+recomendado que você use a mesma versão da imagem base como seu
+servidor Windows de cargas de trabalho, para garantir que você não tenha 
+nenhuma imagem não utilizada ocupando espaço no nó. Contêineres HostProcess 
+também suportam [montagem de volume](#volume-mounts) dentro do volume do contêiner.
+
+### Quando devo usar um contêiner do Windows HostProcess?
+
+- Quando você precisar executar tarefas que exigem a rede do namespace no host.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:42]: "exigem o namespace de rede do host" would be a more faithful translation.

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 11]
```diff
@@ -0,0 +1,245 @@
...
+
+No Kubernetes {{< skew currentVersion >}}, O recurso de contêiner do HostProcess 
+é ativado por padrão. 
+O kubelet irá comunicar-se com o contêiner diretamente passando a flag HostProcess 
+via `CRI`. Você pode usar a versão mais recente do containerd (v1.6+) para executar 
+contêineres HostProcess.
+[Como instalar o containerd.](/docs/setup/production-environment/container-runtimes/#containerd)
+
+Para *desabilitar* contêineres HostProcess você precisa passar a seguinte flag 
+de portal de funcionalidade para o **kubelet** e **kube-apiserver**:
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:67]: Since "feature gate" is a Kubernetes technical term, it would be better to not translate it and instead stylize it as italic, e.g. "você precisa passar o seguinte _feature gate_...".

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 12]
```diff
@@ -0,0 +1,245 @@
...
+
+```powershell
+--feature-gates=WindowsHostProcessContainers=false
+```
+
+Veja a documentação [Portal de Funcionalidade](/docs/reference/command-line-tools-reference/feature-gates/#overview)
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:73]: Same here: "veja a documentação de _feature gates_...".

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 13]
```diff
@@ -0,0 +1,245 @@
...
+para mais detalhes.
+
+
+
+## Limitações
+
+Essas limitações são relevantes para o Kubernetes v{{< skew currentVersion >}}:
+
+- Contêineres HostProcess requerem containerd 1.6 ou mais recente
+  {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:83]: Rewording suggestion: "Contêineres HostProcess requerem o agente de execução de contêiner containerd 1.6 ou mais recente".

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 14]
```diff
@@ -0,0 +1,245 @@
...
+  Nenhum sistema de arquivos ou isolação do Hyper-V são suportados para contêineres 
+  HostProcess.
+- `Volume mounts` são suportados e são montados dentro do volume do contêiner. Veja
+  [Volume Mounts](#volume-mounts)
+- Um conjunto limitado de contas de usuário do host está disponível para os contêineres 
+  HostProcess por padrão.
+  Veja [Escolhendo uma conta de usuário](#choosing-a-user-account).
+- Limites de recursos (disco, memória, número de cpus) são suportados da mesma maneira 
+  que os processos no host.
+- Ambos, `Named pipe mounts` e `Unix domain sockets` **não** são suportados e devem
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:98]: About this sentence:
* "named pipe" has a semi-translation to Portuguese as "pipe nomeado" (see https://pt.wikipedia.org/wiki/Pipe_nomeado).
* "Unix domain socket" also has a translation as "soquete de domínio Unix" (see https://pt.wikipedia.org/wiki/Soquete_de_dom%C3%ADnio_Unix).

How do you feel translating "named pipe mounts" to "montagens de pipe nomeado" and "Unix domain sockets" to "soquetes de domínio Unix"?

The comma after "Ambos" should not exist as it changes the meaning of this sentence.

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 15]
```diff
@@ -0,0 +1,245 @@
...
+  ser acessados através de seu caminho no host (ex. \\\\.\\pipe\\\*)
+
+## Pod HostProcess requisitos de configuração
+
+Habilitar um Pod do Windows HostProcess requer a definição das configurações corretas 
+de segurança na configuração do pod. Das políticas definidas no 
+[Padrões de segurança de pod](/docs/concepts/security/pod-security-standards)
+Pods HostProcess são desabilitados pela linha de base e políticas restritas.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:106]: While your translation for _baseline_ and _restricted_ are very good, I think we should keep the original in English, as they refer to specific configurations.

Suggestion: "Entre as políticas definidas nos padrões de segurança de Pod, os Pods HostProcess não são permitidos nas políticas _baseline_ e _restricted_".

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 16]
```diff
@@ -0,0 +1,245 @@
...
+  ser acessados através de seu caminho no host (ex. \\\\.\\pipe\\\*)
+
+## Pod HostProcess requisitos de configuração
+
+Habilitar um Pod do Windows HostProcess requer a definição das configurações corretas 
+de segurança na configuração do pod. Das políticas definidas no 
+[Padrões de segurança de pod](/docs/concepts/security/pod-security-standards)
+Pods HostProcess são desabilitados pela linha de base e políticas restritas. 
+É, portanto, recomendado que estes pods HostProcess executem em alinhamento 
+com o perfil privilegiado.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:108]: Same consideration here as the previous sentence. I think it would be better to keep _privileged_.

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 17]
```diff
@@ -0,0 +1,245 @@
...
+
+Ao executar sob a política privilegiada, as configurações a seguir precisam 
+ser definidas para permitir a criação de um pod HostProcess:
+
+<table>
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:113]: This entire table is missing translations.

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 18]
```diff
@@ -0,0 +1,245 @@
...
+`$CONTAINER_SANDBOX_MOUNT_POINT\var\run\secrets\kubernetes.io\serviceaccount\`
+
+## Limites de recursos
+
+Limites de recursos (disco, memória, número de cpus) são aplicados ao job todo.
+Por exemplo, com um limite de 10MB definido, a memória alocada para qualquer 
+objeto job HostProcess será limitado em 10MB. 
+Este é o mesmo comportamento que outros tipos de contêiner do Windows.
+Esses limites seriam especificados da mesma maneira que são atualmente, para qualquer 
+orquestrador ou runtime sendo usado. A única diferença está no cálculo do uso
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:217]: I would translate "runtime" to "agente de execução".

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 19]
```diff
@@ -0,0 +1,245 @@
...
+Contêineres HostProcess tem a capacidade de executar como uma das três contas 
+de serviço do Windows suportadas:
+
+- **[LocalSystem](https://docs.microsoft.com/windows/win32/services/localsystem-account)**
+- **[LocalService](https://docs.microsoft.com/windows/win32/services/localservice-account)**
+- **[NetworkService](https://docs.microsoft.com/windows/win32/services/networkservice-account)**
+
+Você deve selecionar uma conta de serviço do Windows apropriada para cada 
+contêiner HostProcess, com o objetivo de limitar o grau de privilégios, 
+a fim de evitar acidentais (ou até maliciosos) danos ao host. O serviço de conta
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:232]: I would rephrase this as "evitar danos acidentais (ou até mesmo maliciosos) ao host".

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 20]
```diff
@@ -0,0 +1,245 @@
...
+use a conta do serviço `LocalService`, pois é o menos privilegiado das três opções.
+
+## Solução de problemas de contêineres de HostProcess
+
+- Contêineres HostProcess falham na inicialização com `failed to create user process token: failed to logon user: Access is denied.: unknown`
+
+  Verifique se o contêiner está executando como a conta de serviço 
+  `LocalSystem` ou `LocalService`. Contas de usuário (mesmo contas de administrador) 
+  não tem permissões para criar tokens de logon para qualquer uma das  
+  [contas de usuários](#choosing-a-user-account) suportadas.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:244]: Minor: the indentation before `contas de usuários` is breaking rendering.

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @DonatoHorn! 👋

Thank you very much for addressing the previous feedback. 🎉

I apologize for not doing a new review earlier, as I unfortunately missed your latest commit.

I found some more bits that need to be addressed for readability. Would you feel comfortable making those changes so we can move your contribution forward?

If you disagree with any of those bits, have a different suggestion, want to discuss the review, or need some help addressing the feedback, please feel free to reach out to us on the `#kubernetes-docs-pt` channel of Kubernetes Slack. We're available to help and answer any questions you may have. 😃

Thanks again for your work, and I look forward to having this contribution merged!

**Comment by stormqueen1990** [comment]
/retitle [pt-br] Add docs/tasks/configure-pod-container/create-hostprocess-pod.md

**Comment by stormqueen1990** [comment]
/cc @MrErlison @edsoncelio
Could you please provide a second review?

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 21]
```diff
@@ -0,0 +1,245 @@
+---
+title: Crie um Pod de Hostprocess do Windows
```
Comment (MrErlison) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:2]: "HostProcess" instead of "Hostprocess" for coherence. line 12

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 22]
```diff
@@ -0,0 +1,245 @@
...
+cargas de trabalho em um host Windows contêinerizado. Esses contêineres operam como
+processos normais, mas tem acesso à rede host do namespace,
+armazenamento e dispositivos, quando recebem os privilégios de usuário apropriados.
+Os contêineres HostProcess podem ser usados para implantar plugins de rede,
+configurações de armazenamento, plugins de dispositivos, kube-proxy, e outros
+componentes para nós do Windows, sem a necessidade de *proxies* dedicados ou
+a instalação direta dos serviços de host.
+
+Tarefas administrativas, como a instalação de `patches` de segurança, 
+coleta de logs de eventos, e mais pode ser realizado sem a necessidade
```
Comment (MrErlison) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:22]: "podem ser realizadas" instead of "pode ser realizado" because "Tarefas administrativas podem ser realizadas"

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 23]
```diff
@@ -0,0 +1,245 @@
...
+Essas limitações são relevantes para o Kubernetes v{{< skew currentVersion >}}:
+
+- Contêineres HostProcess requerem containerd 1.6 ou mais recente
+  {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
+- Pods HostProcess só podem conter contêineres HostProcess. Esta é uma limitação atual
+  do sistema operacional Windows; Contêineres do Windows sem privilégios não podem 
+  compartilhar um vNIC com o IP do host no namespace.
+- Contêineres HostProcess executam como um processo do host e não tem nenhum grau de
+  isolamento, além das restrições de recursos impostas à conta de usuário do HostProcess. 
+  Nenhum sistema de arquivos ou isolação do Hyper-V são suportados para contêineres
```
Comment (MrErlison) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:89]: Sounds better to me "Nem o sistema de arquivos nem o isolamento Hyper-V" instead of "Nenhum sistema de arquivos ou isolação do Hyper-V". Think about it.

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 24]
```diff
@@ -0,0 +1,245 @@
...
+  compartilhar um vNIC com o IP do host no namespace.
+- Contêineres HostProcess executam como um processo do host e não tem nenhum grau de
+  isolamento, além das restrições de recursos impostas à conta de usuário do HostProcess. 
+  Nenhum sistema de arquivos ou isolação do Hyper-V são suportados para contêineres 
+  HostProcess.
+- `Volume mounts` são suportados e são montados dentro do volume do contêiner. Veja
+  [Volume Mounts](#volume-mounts)
+- Um conjunto limitado de contas de usuário do host está disponível para os contêineres 
+  HostProcess por padrão.
+  Veja [Escolhendo uma conta de usuário](#choosing-a-user-account).
```
Comment (MrErlison) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:95]: The link is broken.

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 25]
```diff
@@ -0,0 +1,245 @@
...
+- Limites de recursos (disco, memória, número de cpus) são suportados da mesma maneira 
+  que os processos no host.
+- Ambos, `Named pipe mounts` e `Unix domain sockets` **não** são suportados e devem
+  ser acessados através de seu caminho no host (ex. \\\\.\\pipe\\\*)
+
+## Pod HostProcess requisitos de configuração
+
+Habilitar um Pod do Windows HostProcess requer a definição das configurações corretas 
+de segurança na configuração do pod. Das políticas definidas no
```
Comment (MrErlison) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:104]: minor: pod -> Pod

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 26]
```diff
@@ -0,0 +1,245 @@
...
+
+Habilitar um Pod do Windows HostProcess requer a definição das configurações corretas 
+de segurança na configuração do pod. Das políticas definidas no 
+[Padrões de segurança de pod](/docs/concepts/security/pod-security-standards)
+Pods HostProcess são desabilitados pela linha de base e políticas restritas. 
+É, portanto, recomendado que estes pods HostProcess executem em alinhamento 
+com o perfil privilegiado.
+
+Ao executar sob a política privilegiada, as configurações a seguir precisam 
+ser definidas para permitir a criação de um pod HostProcess:
```
Comment (MrErlison) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:111]: minor: pod -> Pod

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 27]
```diff
@@ -0,0 +1,245 @@
...
+orquestrador ou runtime sendo usado. A única diferença está no cálculo do uso 
+de recursos de disco usados para rastreamento de recursos, devido à diferença em 
+como contêineres HostProcess são inicializados.
+
+## Escolha uma conta de usuário
+
+Contêineres HostProcess tem a capacidade de executar como uma das três contas 
+de serviço do Windows suportadas:
+
+- **[LocalSystem](https://docs.microsoft.com/windows/win32/services/localsystem-account)**
```
Comment (MrErlison) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:226]: There is the article in pt-br. What do you think about update the link?
https://learn.microsoft.com/pt-br/windows/win32/services/localsystem-account

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 28]
```diff
@@ -0,0 +1,245 @@
...
+- **[LocalService](https://docs.microsoft.com/windows/win32/services/localservice-account)**
```
Comment (MrErlison) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:227]: There is the article in pt-br. What do you think about update the link?
https://learn.microsoft.com/pt-br/windows/win32/services/localservice-account

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 29]
```diff
@@ -0,0 +1,245 @@
...
+- **[NetworkService](https://docs.microsoft.com/windows/win32/services/networkservice-account)**
```
Comment (MrErlison) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:228]: There is the article in pt-br. What do you think about update the link?
https://learn.microsoft.com/pt-br/windows/win32/services/networkservice-account

File: content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md (hunk 20 above)
Comment (MrErlison) [inline content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md:244]: The link is broken.

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @DonatoHorn! 👋 

Thank you very much for your work in this pull request. I left a few more comments for accuracy. If you disagree with anything, please feel free to reach out to me on the `#kubernetes-docs-pt` channel in Kubernetes Slack. 😃

I've noticed that this page is currently outdated. I suggest we get it merged with the content as-is after this last set of corrections, and then we can open an issue to bring the content up-to-date.

Thanks again! 🎉 

---

Olá, @DonatoHorn! 👋

Muito obrigada pelo teu trabalho neste pull request. Deixei mais alguns comentários de pontos que necessitam correção para precisão na tradução. Se discordares de algo, podes entrar em contato comigo no canal `#kubernetes-docs-pt` no Slack do Kubernetes. 😃

Notei que esta página está um pouco desatualizada com a versão atual. Sugiro que nos foquemos em mesclar a tradução como está após esta última iteração de correções, e na sequência podemos abrir uma issue para atualizar o conteúdo com a última versão em inglês.

Obrigada novamente! 🎉

**Comment by seokho-son** [comment]
Hi @stormqueen1990 
Could you check status of this PR? 
Do we require additional reviews, or should we consider this PR as stale?

**Comment by stormqueen1990** [comment]
> Hi @stormqueen1990 Could you check status of this PR? Do we require additional reviews, or should we consider this PR as stale?

Hi there, @seokho-son! I'll take a look. This one slipped under my radar 😅

**Comment by stormqueen1990** [comment]
Hi there, @DonatoHorn!

Thanks for your work in this translation!

Apologies for not getting back to this PR earlier. I took a look, and it appears the content is already outdated. Because of that, I will be closing this PR, but feel free to update the content and reopen it, or create a separate PR with the updated content.

/close
